packaging/hudi-utilities-slim-bundle/pom.xml

```diff
@@ -92,9 +92,7 @@
                 <includes>
                   <include>org.apache.hudi:hudi-common</include>
                   <include>org.apache.hudi:hudi-client-common</include>
-                  <include>org.apache.hudi:hudi-spark-client</include>
                   <include>org.apache.hudi:hudi-utilities_${scala.binary.version}</include>
-                  <include>org.apache.hudi:hudi-hive-sync</include>
                   <include>org.apache.hudi:hudi-sync-common</include>
```

Comment:
sync-common should also be removed too? by right, the user-chosen metasync bundle should have it

Reply:
yes this should be removed as well. I have removed it and verified with`hudi-hive-sync-bundle`. working fine.